tests/test_sklearn_bagging.py

```diff
@@ -71,7 +71,7 @@ def test_bagging_linear_svr(self):
         hb_model = hummingbird.ml.convert(reg, "torch")
 
         np.testing.assert_allclose(
-            reg.predict(X), hb_model.predict(X), rtol=1e-05, atol=1e-05,
+            reg.predict(X), hb_model.predict(X), rtol=1e-04, atol=1e-04,
```

Comment:
Let's document this somewhere; maybe in pydoc?

Reply:
I added a comment on `_parse`